azurelinuxagent/common/protocol/wire.py

```diff
@@ -1000,12 +1053,14 @@ def report_role_prop(self, thumbprint):
 
     def report_health(self, status, substatus, description):
         goal_state = self.get_goal_state()
+        users = self.get_local_users()
```

Comment:
There is no limit on this value.  What happens if this value is large?  Should this value be reported for all VMs?  I think not, and I am concerned that you are.

Reply:
I'll pull out any reporting logic.  I'm not sure why Pawel added this as I've verified with our PM there's no requirement for it from our side.